student.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Student < Person
+  def initialize(classroom)
+    super()
```

Comment:
- [ ] Constructor should extend the parent's constructor by adding @classroom and a parameter for it. please have a look at [this link ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars)for an example